src/forms/timeInput.test.jsx

```diff
@@ -1,58 +1,77 @@
+
 import React from 'react';
-import TestUtils from 'react-addons-test-utils';
+import { shallow } from 'enzyme';
 import TimeInput from './TimeInput';
 
 describe('TimeInput', function() {
 
-	let timeInputComponent,
-		timeInputEl;
+	let component,
+		inputEl;
 
 	const timeValue = '22:00',
-		newTime = '23:00',
-		callbackSpy = jest.fn();
+		newTime = '23:00';
+
+	const callbackSpy = jest.fn(),
+		onChangeMock = jest.fn();
 
 	beforeEach(() => {
-		timeInputComponent = TestUtils.renderIntoDocument(
-			<TimeInput name='time' value={timeValue} onChangeCallback={callbackSpy} required />);
-		timeInputEl = TestUtils.findRenderedDOMComponentWithTag(timeInputComponent, 'input');
+		component = shallow(
+			<TimeInput
+				name='time'
+				value={timeValue}
+				onChange={onChangeMock}
+				required />
+		);
+		inputEl = component.find('input');
 	});
 
 	afterEach(() => {
-		timeInputComponent = null;
-		timeInputEl = null;
+		component = null;
+		inputEl = null;
 	});
 
-	it('exists', function() {
-		expect(timeInputEl).not.toBeNull();
+	it('renders a time html input with expected props', function() {
+		expect(component).toMatchSnapshot();
 	});
 
 	it('takes a value in HH:mm format', function() {
```

Comment:
the snapshot is explicit about this case, so you shouldn't need a separate unit test for it.